src/styles/card.scss

```
@@ -0,0 +1,24 @@
+.card {
+  display: block;
+  width: 198px;
+  background-color: var(--white-color);
+  border: 1px solid var(--darck_white-color);
+  border-radius: 5px;
+  justify-content: center;
+  align-items: center;
+
+  & .image {
+    padding: 32px 19px 36px;
```

margin to get outer indent
```suggestion
    padding: 0 auto 40px;
```